<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClickScrollingTestWidget</class>
 <widget class="QMainWindow" name="ClickScrollingTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ClickScrollingTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>800</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>800</height>
      </size>
     </property>
     <widget class="QPushButton" name="buttonHiddenByOverflow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>buttonHiddenByOverflow</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="visibleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>visibleButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonHiddenByOverflow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>radioButtonHiddenByOverflow</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxHiddenByOverflow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>221</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>checkBoxHiddenByOverflow</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="visibleRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>visibleRadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="partiallyVisibleRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>partiallyVisibleRadioButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="outOfViewButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>outOfViewButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
